public/electron.js

```diff
@@ -157,7 +158,8 @@ const generateMenu = () => {
           label: 'About',
           click: () => {
             openAboutWindow({
-              icon_path: ICON_PATH,
+              base_path: resolvePath(app.getAppPath()),
+              icon_path: path.join(__dirname, ICON_PATH),
```

Comment:
For example, here I think Node will normalise it for Windows. But still, I'm not sure if we also need to have the appPath here instead of using `path.join`.

Reply:
I tried multiple combinaisons, but at the end once I run `yarn dist` the only solution which still provide the icon image is this one. I think it has to do with how the final files are built and where they are located then.

Reply:
OK. I still find it a bit weird that there is not a simpler solution. Let's review it on Friday.